### PR TITLE
comic-mandown: init at 1.5.0

### PR DESCRIPTION
**Pull request by Scrumplex**
###### Description of changes

- python3Packages.ebooklib: init at 0.18
- python3Packages.comicon: init at 1.0.0
- comic-mandown: init at 1.5.0

Mandown is a comic downloader and a CBZ, EPUB, MOBI, and/or PDF converter.

https://github.com/potatoeggy/mandown

mandown needs comicon: https://github.com/potatoeggy/comicon

comicon needs ebooklib: https://github.com/aerkalov/ebooklib

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Scrumplex** [comment]
I might need some input on the package name. We already have a package called `mandown`. I am not sure if `comic-mandown` is a good name here.

**Comment by wegank** [comment]
Repology classifies this as [`mandown-comics`](https://repology.org/project/mandown-comics/versions), but `comic-mandown` looks fine.

**Comment by Scrumplex** [comment]
> Repology classifies this as [`mandown-comics`](https://repology.org/project/mandown-comics/versions), but `comic-mandown` looks fine.

I would be fine with Repology's name too. I just came up with something when I noticed that we already have another `mandown`

**Comment by Scrumplex** [comment]
I have moved `mandown` into a python package, so that we can use `toPythonApplication` instead.

`mandown` does have a basic interface for libraries too.

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -4890,6 +4890,8 @@ with pkgs;
     inherit (darwin.apple_sdk.frameworks) Security;
   };
 
+  comic-mandown = callPackage ../tools/misc/comic-mandown { };
```
Comment (wegank) [inline pkgs/top-level/all-packages.nix:4893]: The package can then be moved to `pkgs/by-name`.

**Review by wegank — APPROVED** [review]

**Comment by wegank** [comment]
Result of `nixpkgs-review pr 245353` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>14 packages built:</summary>
  <ul>
    <li>comic-mandown</li>
    <li>comic-mandown.dist</li>
    <li>python310Packages.comicon</li>
    <li>python310Packages.comicon.dist</li>
    <li>python310Packages.ebooklib</li>
    <li>python310Packages.ebooklib.dist</li>
    <li>python310Packages.mandown</li>
    <li>python310Packages.mandown.dist</li>
    <li>python311Packages.comicon</li>
    <li>python311Packages.comicon.dist</li>
    <li>python311Packages.ebooklib</li>
    <li>python311Packages.ebooklib.dist</li>
    <li>python311Packages.mandown</li>
    <li>python311Packages.mandown.dist</li>
  </ul>
</details>

**Pull request merged**